pyhub/mcptools/microsoft/excel/com/decorators.py

```
@@ -0,0 +1,184 @@
+            com_initialized = True
+        except:
+            # COM was already initialized
+            pass
+
+        try:
+            return func(*args, **kwargs)
+        finally:
+            if com_initialized:
+                try:
+                    pythoncom.CoUninitialize()
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)